bullet/client.py

```diff
@@ -131,16 +131,24 @@ def __init__(
             raise ValueError("Indent must be > 0!")
         if margin < 0:
             raise ValueError("Margin must be > 0!")
-
+        
         self.prompt = prompt
-        self.choices = choices
+        
+        #test flag also return objects
+        self.objects=[]
+        if(type(choices) is dict):
```

Comment:
`isinstance(obj, cls)` is generally preferable to `type(obj) is cls` since it supports subtypes. (For instance, `OrderedDict`)